sickbeard/notifiers/trakt.py

```diff
@@ -105,6 +105,87 @@ def update_library(self, ep_obj):
             except (traktException, traktAuthException, traktServerBusy) as e:
                 logger.log(u"Could not connect to Trakt service: %s" % ex(e), logger.WARNING)
 
+    def update_watchlist (self, show_obj, s=None, e=None, data_obj=None, update="add"):
+
+        """
+        Sends a request to trakt indicating that the given episode is part of our library.
+
+        ep_obj: The TVEpisode object to add to trakt
+        """
+
+        trakt_id = sickbeard.indexerApi(show_obj.indexer).config['trakt_id']
+        trakt_api = TraktAPI(sickbeard.TRAKT_API_KEY, sickbeard.TRAKT_USERNAME, sickbeard.TRAKT_PASSWORD)
+
+        logger.log(u"update: " + str(update), logger.ERROR)
+
+        if sickbeard.USE_TRAKT:
```

Comment:
This `if` is completely unnecessary. We already know they are `USE_TRAKT` if this method has been called.